docker/Dockerfile

```diff
@@ -2,6 +2,10 @@
 # triggered the rebuild of dependent containers.
 FROM python:3.6-slim
 
+RUN apt-get update -y && \
```

Comment:
I think the `-y` for `apt-get update` might not be required, but it probably doesn't matter too much...